<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_8">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QWidget" name="widget_5" native="true">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,0,0,2">
          <item>
           <widget class="QLabel" name="label_ip">
            <property name="text">
             <string>IP地址:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_ip"/>
          </item>
          <item>
           <widget class="QPushButton" name="button_link">
            <property name="styleSheet">
             <string notr="true">QPushButton{background-color:rgb(255,255,255);
            color: black;  
			/*border-radius: 5px;*/
	        border: 2px solid grey;
			border-style: outset;
}
QPushButton:hover{
background-color:rgb(225,225,225); 
color: blue;

}
QPushButton:pressed{background-color:rgb(210,210,210); border-style: inset; }</string>
            </property>
            <property name="text">
             <string>连接</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>478</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0">
         <item>
          <widget class="QWidget" name="widget" native="true">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="leftMargin">
             <number>1</number>
            </property>
            <property name="topMargin">
             <number>1</number>
            </property>
            <property name="rightMargin">
             <number>1</number>
            </property>
            <property name="bottomMargin">
             <number>15</number>
            </property>
            <item>
             <widget class="QTableWidget" name="tableWidget">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="styleSheet">
               <string notr="true">QHeaderView       
{             
    background-color:gray;      
}
QHeaderView::section 
{          
    font-size:14px;               
    font-family:&quot;Microsoft YaHei&quot;;
    color:#000000;               
    background:rgb(225,225,225);          
    border: none;
 
    min-height:30px;
    margin-left:0px;             
    padding-left:0px;             
}
QTableWidget               
{
    background-color: rgb(255, 255, 255);     
    border:none;              
    font-size:12p;
    color:#666666;             
}

QTableWidget::item::selected    
{                 
    background: rgba(0, 68, 238,50%);         
}

QScrollBar::handle:vertical   
{     
    background-color:rgb(0, 68, 238);                            
    border: 0px solid grey;          
    border-radius:3px;               
    width: 8px;                  
}
QScrollBar::vertical                   
{
    border-width:1px;                       
    border-style: solid;                  
    border-color: rgba(255, 255, 255, 10%); 
    width: 8px;                        
    margin:0px 0px 0px 0px;                
    border-radius:3px;                     
}

QScrollBar::add-page:vertical
{
    background-color:rgb(255,255,255); 
}
QScrollBar::sub-page:vertical
{
	 background-color:rgb(255,255,255); 
}
QScrollBar::sub-line:vertical 
{

	
	border-color: rgb(255, 255, 255);
    background:transparent; 
}
QScrollBar::add-line:vertical
{

   
	border-color: rgb(255,255,255);
  	background:transparent;
}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="midLineWidth">
               <number>0</number>
              </property>
              <property name="editTriggers">
               <set>QAbstractItemView::NoEditTriggers</set>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <property name="gridStyle">
               <enum>Qt::SolidLine</enum>
              </property>
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>20</number>
              </attribute>
              <attribute name="horizontalHeaderHighlightSections">
               <bool>true</bool>
              </attribute>
              <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
               <bool>false</bool>
              </attribute>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>true</bool>
              </attribute>
              <attribute name="verticalHeaderVisible">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string>COMMAND</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_3" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="topMargin">
             <number>13</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="button_all">
                <property name="styleSheet">
                 <string notr="true">QPushButton{background-color:rgb(255,255,255);
            color: black;  
			/*border-radius: 5px;*/
	        border: 2px solid grey;
			border-style: outset;
}
QPushButton:hover{
background-color:rgb(225,225,225); 
color: blue;

}
QPushButton:pressed{background-color:rgb(210,210,210); border-style: inset; }
</string>
                </property>
                <property name="text">
                 <string>全部执行</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>118</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="4,0">
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>15</number>
            </property>
            <item>
             <widget class="QTextBrowser" name="textBrowser">
              <property name="styleSheet">
               <string notr="true">QTextBrowser
{
	background-color:rgb(255,255,255);
     font-size:13px;               
     font-family:&quot;Microsoft YaHei&quot;;                        
}
QScrollBar::handle:vertical   
{     
    background-color:rgb(0, 68, 238);                            
    border: 0px solid grey;          
    border-radius:3px;               
    width: 8px;                  
}
QScrollBar::vertical                   
{
    border-width:1px;                       
    border-style: solid;                  
    border-color: rgba(255, 255, 255, 10%); 
    width: 8px;                        
    margin:0px 0px 0px 0px;                
    border-radius:3px;                     
}

QScrollBar::add-page:vertical
{
    background-color:rgb(255,255,255); 
}
QScrollBar::sub-page:vertical
{
	 background-color:rgb(255,255,255); 
}
QScrollBar::sub-line:vertical 
{

	
	border-color: rgb(255, 255, 255);
    background:transparent; 
}
QScrollBar::add-line:vertical
{

   
	border-color: rgb(255,255,255);
  	background:transparent;
}</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="Label_msgPush">
                <property name="text">
                 <string>发送消息:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_send">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>128</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="label_msgRecv">
                <property name="text">
                 <string>接收消息:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_recv">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>248</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="button_clear">
                <property name="maximumSize">
                 <size>
                  <width>1677777</width>
                  <height>1677777</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{background-color:rgb(255,255,255);
            color: black;  
			/*border-radius: 5px;*/
	        border: 2px solid grey;
			border-style: outset;
}
QPushButton:hover{
background-color:rgb(225,225,225); 
color: blue;

}
QPushButton:pressed{background-color:rgb(210,210,210); border-style: inset; }</string>
                </property>
                <property name="text">
                 <string>清空显示</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
